src/main.ts

```diff
@@ -187,7 +187,7 @@ app.whenReady().then(
     nativeTheme.themeSource = k6StudioState.appSettings.appearance.theme
     await createSplashWindow()
 
-    await sendReport(k6StudioState.appSettings.telemetry.usageReport)
+    UsageTracker.getInstance()
```

Comment:
Currently, `getInstance` initializes the service if this hasn't happened yet, which is exactly how it's used here. An alternative would be having both `initialize()` and `getInstance()` - wdyt?